test/ui-test/src/screens/MainUi.py

```diff
@@ -0,0 +1,23 @@
+# ******************************************************************************
+# Status.im
+# *****************************************************************************/
+# /**
+# * \file    MainUi.py
```

Comment:
We already have StatusMainScreen

Reply:
Thanks for the heads up. This was meant to be for a generic global helper functionality but I now realize that maybe is premature optimizaiton.